parsers/DK.py

```diff
@@ -30,6 +30,11 @@ def fetch_data(
     """
     ses = session or Session()
 
+    if target_datetime.tzinfo:
+        # Data source doesn't support timezone aware
+        # datetimes.
+        target_datetime.replace(tzinfo=None)
```

Comment:
```suggestion
        target_datetime = target_datetime.replace(tzinfo=None)
```

See [here](https://www.geeksforgeeks.org/datetime-replace-function-in-python/) for documentation that explains that `.replace` returns a datetine and does not do inplace mutation

Weirdly it seems that it fixes real-time but then it throws an internal server error with historical data:

https://api.energidataservice.dk/dataset/ElectricityProdex5MinRealtime?limit=144&filter=%7B%22PriceArea%22%3A%22DK1%22%7D&start=2022-12-29T00%3A00&end=2022-12-30T00%3A00

Reply:
At the same time the endpoint is called "RealTime" so maybe that's to be expected 😅

Reply:
I'm not sure why but now the historical data link you posted seems to work just fine...